### ISSUE TITLE
Create a label to expedite PR reviews

### ISSUE THREAD

**Issue by fejta** [kubernetes/test-infra#1741]
See https://groups.google.com/d/msg/kubernetes-wg-contribex/Q9aXr15Wpf0/KTjoXGdlBwAJ

TLDR;

Seal Team Six for PRs

We can call it whatever the heck we want, but the Seals, and other Special Forces organizations, know how to get stuff done.  We have a Dam that needs to be drained, without harming our community or our product.  I am not saying that we will drain the Dam, but we can be strategic, short term.

1. Create a team modelling what the Seals / Special Forces do
2. The teams mandate is to triage and get stuff done.  80 / 20 rule  
3. Help with the optics, we do the work, and prove it works.  Then we let the community know what we are doing
4. Help with the optics, tell the community what we are doing in regards to breaking apart core.  

---

one way we can codify your proposal:
* Create a seal-team-six, i-can-haz-review, or some similarly-named label.
* Create a prow plugin that allows people to flag a PR that is not getting attention quickly
    * Call it something like /expedite, /this-one-needs-11, /ludicrous-speed, /batsignal, /omg-someone-review-my-fscking-pr /i-like-turtles, whatever. Kudos for making it charming and fun.
* Update gubernator to become aware of the seal-team label like we do needs rebase and/or release-note-label-needed
    * Store the list of people who are on seal team six somewhere that gubernator can find, or alternatively allow anyone to enlist in the seal team
    * Expect the seal team to regularly check the PR dashboard for PRs that need routing attention.

If it seems like this is working well, adding an /assign plugin to prow will expand the set of people who can assign PRs to others.

Prow plugins live here: https://github.com/kubernetes/test-infra/tree/master/prow/plugins
Gubernator lives here: https://github.com/kubernetes/test-infra/tree/master/gubernator

The idea is something along the lines of this showing up as a new "Deploy the Seals" or whatever section in the PR Dashboard -- https://k8s-gubernator.appspot.com/pr/
